docs/source/reference/config.rst

```diff
@@ -708,6 +710,30 @@ Supported values:
       - sky-serve-controller-*: my-service-account-2
       - "*": my-default-service-account
 
+.. _config-yaml-aws-post-provision-runcmd:
+
+``aws.post_provision_runcmd``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run commands during the instance initialization phase (optional).
+
+This is useful for doing any setup that must happen right after the instance starts, such as:
+
+- Configuring system settings
+- Installing certificates
+- Installing packages
+
```

Comment:
we can mention that this will be exectued through cloud init's runcmd?

Reply:
good idea, added!